### PR TITLE
Enable main wallet screen. WIP features are hidden behind flag

### PR DESCRIPTION
**Pull request by jeluard**
### Summary:

Enable main wallet screen in all builds.

### Steps to test:
- Verify Wallet tab is enabled
- Open Wallet tab
- All icons should be actionable but show `TODO` popup

status: ready

### CODE REVIEW

File: .env.jenkins [hunk 1]
```diff
@@ -1,2 +1 @@
-TESTFAIRY_ENABLED=1
-WALLET_TAB_ENABLED=1
```
Comment (oskarth) [inline .env.jenkins]: Can we make this explicit to `WALLET_WIP_ENABLED=0`?

**Review by oskarth — APPROVED** [review]
Looks good to me

File: .env.jenkins (hunk 1 above)
Comment (jeluard) [reply .env.jenkins]: To have an explicit list of all flags I guess?

Comment (oskarth) [reply .env.jenkins]: Yes

**Comment by asemiankevich** [comment]
Tested both iOS / Android, looks good to me

**Pull request merged**